glitter/blocks/image/managers.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class BaseImageBlockManager(models.Manager):
+    use_for_related_fields = True
+
+    def get_queryset(self):
+        qs = super(BaseImageBlockManager, self).get_queryset()
+        return qs.select_related('image')
```

Comment:
`use_for_related_fields` - activates the manager for related querysets (otherwise it only gets used with `ImageBlock.objects.method()`).

And we have to specify the field name as image is a null field, `select_related()` without any arguments won't select null fields by default.